invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html

```diff
@@ -11,6 +11,13 @@
 
 {% if use_theme_basic_template|default(true) %}
   {%- extends config.BASE_TEMPLATE %}
+  {%- block css %}
+    {{ super() }}
+    {% if community %}
```

Comment:
see https://github.com/inveniosoftware/invenio-app-rdm/pull/2552/files#diff-7ff1cc8443dcfeeb711849c766e33d7d8d5d785fee0d982bf849e060ff2500a9R4
we should decide which variable we use, sometimes we use community, sometimes community_theme, it might bring confusion